spring-cloud-data-module-deployers/spring-cloud-data-module-deployer-local/src/main/java/org/springframework/cloud/data/module/deployer/local/LocalModuleDeployer.java

```diff
@@ -77,9 +79,11 @@ public ModuleDeploymentId deploy(ModuleDeploymentRequest request) {
 		catch (Exception e) {
 			throw new IllegalStateException("failed to determine URL for module: " + module, e);
 		}
-		args.add(String.format("--%s.server.port=%d", module, port));
-		args.add(String.format("--%s.endpoints.shutdown.enabled=true", module));
-		launcher.launch(new String[] { module }, args.toArray(new String[args.size()]));
+		args.put("server.port", String.valueOf(port));
```

Comment:
Sorry, I realize that now. We are constructing a dedicated launch request instance per module in this case. In fact I believe this was the original rationale for creating that class.